test/screenshot/lib/cli-arg-parser.js

```diff
@@ -162,6 +175,11 @@ E.g., 'origin/master' (default), 'HEAD', 'feat/foo/bar', 'fad7ed3:path/to/golden
     return this.args_['mdc_diff_base'];
   }
 
+  /** @return {boolean} */
+  get skipBuild() {
+    return this.args_['mdc_skip_build'];
```

Comment:
Silly question but how/where is `mdc_skip_build` defined as a boolean?

Reply:
Not a silly question at all!

The `argparse` API doesn't make it very clear, but the param is defined as a boolean here:

https://github.com/material-components/material-components-web/pull/2773/files/c2f2b6b5a6871b5d5c3ca84eec51a493c9a3b220#diff-d2bba4088f19b97ee4fe618d0353157aR131

The `action: 'storeTrue'` line the argument config sets the value to `true` if the argument is present, or `false` if it's omitted.

Is there a way I could make this clearer?

Reply:
Probably, but that's mostly on `argparse` for having an unclear API. I'd add a comment explaining that it casts as a boolean.

Reply:
Done!